README.md

```diff
@@ -21,6 +21,8 @@ This is a personal exercise on moving features from [go.nvim](https://github.com
 
 # Installation
 
+Use at least a neovim stable version.
```

Comment:
If we have `VIM_VERSION_MINOR_SUPPORTED = 7` in the Lua file, why not add something like that here?

```suggestion
Requires Neovim 0.7+.
```